pkg/gencmd/util/string.go

```diff
@@ -0,0 +1,33 @@
+package util
+
+import (
+	"strings"
+
+	"github.com/jinzhu/inflection"
+	"github.com/serenize/snaker"
+)
+
+type String struct {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported type String should have comment or be unexported